src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxconverter.py

```diff
@@ -79,11 +79,13 @@ def __init__(
             if "nidq" in stream:
                 file_path = folder_path / f"{folder_path.stem}_t0.nidq.bin"
                 interface = SpikeGLXNIDQInterface(file_path=file_path)
-            data_interfaces.update({stream: interface})
+            data_interfaces.update({str(stream): interface})  # Without str() casting, is a numpy string
 
         super().__init__(data_interfaces=data_interfaces, verbose=verbose)
 
     def get_conversion_options_schema(self) -> dict:
-        return {
-            name: interface.get_conversion_options_schema() for name, interface in self.data_interface_objects.items()
-        }
+        conversion_options_schema = super().get_conversion_options_schema()
+        conversion_options_schema["properties"].update(
+            {name: interface.get_conversion_options_schema() for name, interface in self.data_interface_objects.items()}
+        )
```

Comment:
@h-mayorquin This is the main diff

Reply:
Ok, this makes sense, I am thinking about the structure of the tiff analogous, do you think it would make sense to follow the logic from here:

https://github.com/catalystneuro/neuroconv/blob/1efe156ef005921c623672b8815eabd3021f6bf7/src/neuroconv/datainterfaces/ophys/brukertiff/brukertiffconverter.py#L28-L30